types/artifacts.d.ts

```diff
@@ -213,6 +215,24 @@ declare global {
         requestId: string | null
       }
 
+      export type RawSourceMap = {
```

Comment:
can you add docstrings from https://sourcemaps.info/spec.html#h.qz3o9nc69um5

Reply:
I would guess that @brendankenny intended the docstrings to go on each property rather than a link to the docs, but I could be wrong 😆